crates/ra_assists/src/handlers/change_return_type_to_result.rs

```diff
@@ -0,0 +1,946 @@
+use ra_syntax::{
+    ast, AstNode,
+    SyntaxKind::{COMMENT, WHITESPACE},
+    SyntaxNode, TextSize,
+};
+
+use crate::{Assist, AssistCtx, AssistId};
+use ast::{BlockExpr, Expr, LoopBodyOwner};
+
+// Assist: change_return_type_to_result
+//
+// Change the function's return type to Result.
+//
+// ```
+// <|>fn foo() -> i32 { 42i32 }
+// ```
+// ->
+// ```
+// fn foo() -> Result<i32, > { Ok(42i32) }
```

Comment:
```suggestion
// fn foo() -> Result<i32, <|>> { Ok(42i32) }
```

Reply:
no as we can see here https://github.com/rust-analyzer/rust-analyzer/blob/master/crates/ra_assists/src/handlers/add_derive.rs#L21 for example the cursor should not be displayed in this documentation